lib/components/fields/array.js

```diff
@@ -27,6 +27,9 @@ export default createReactClass({
     var lookups = [];
 
     var items = this.props.field.value;
+    if (!Array.isArray(items)) {
+      items = [items];
```

Comment:
I wonder if we should check if `items == null` and in that case make an empty array? I just think `[null]` and `[undefined]` wouldn't make much sense.